source/macos/AdaptiveCards/AdaptiveCards/Views/ACRFactTextField.swift

```diff
@@ -56,9 +58,19 @@ class ACRFactTextField: NSView {
         set { labelTextField.textColor = newValue }
     }
     
-    var textValue: String? {
+    var plainTextValue: String? {
         get { return labelTextField.stringValue }
-        set { labelTextField.stringValue = newValue ?? "" }
+        set {
+            labelTextField.allowsEditingTextAttributes = false
+            labelTextField.stringValue = newValue ?? ""
+        }
+    }
+    var attributedTextValue: NSAttributedString? {
+        get { return labelTextField.attributedStringValue }
+        set {
+            labelTextField.allowsEditingTextAttributes = true
```

Comment:
should we set ``` labelTextField.allowsEditingTextAttributes = true ``` it in this or in the render function, just asking

Reply:
It should ideally be here as the render function doesn't know that it's a text field, or it even has an xib.
So we shouldn't expose the properties of it

Reply:
got it